llama-index-integrations/vector_stores/llama-index-vector-stores-deeplake/tests/test_vector_stores_deeplake.py

```diff
@@ -1,16 +1,20 @@
+import os
+
 import jwt  # noqa
 import pytest
-
 from llama_index.core import Document
 from llama_index.core.vector_stores.types import (
     BasePydanticVectorStore,
-    MetadataFilter,
-    MetadataFilters,
     FilterCondition,
     FilterOperator,
+    MetadataFilter,
+    MetadataFilters,
 )
 from llama_index.vector_stores.deeplake import DeepLakeVectorStore
 
+if os.getenv("GITHUB_ACTIONS") == "true":
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables